docs/How-to-run-blackbox-testing.md

```diff
@@ -0,0 +1,437 @@
+DELETE edgex-core-data:48080/api/v1/event/id/57e866d4e4b0ca8e6d73a412 [404 Not Found, 173B, 3ms]
+✓  Status code is 404
+✓  Response time is less than 800
+
+↳ 18 http://localhost:48080/api/v1/event/device/:deviceId/{limit}
+GET edgex-core-data:48080/api/v1/event/device/57e866d4e4b0ca8e6d73a412/100 [200 OK, 110B, 2ms]
+✓  Response time is less than 800
+
+↳ 100 http://localhost:48080/api/v1/event/device/:deviceId
+DELETE edgex-core-data:48080/api/v1/event/device/preassureSensor [200 OK, 108B, 2ms]
+✓  Response time is less than 800
+
+↳ 22 http://localhost:48080/api/v1/event/device/:deviceId
+DELETE edgex-core-data:48080/api/v1/event/device/57e866d4e4b0ca8e6d73a412 [200 OK, 108B, 2ms]
+✓  Response time is less than 800
+
+┌─────────────────────────┬──────────┬──────────┐
+│                         │ executed │   failed │
+├─────────────────────────┼──────────┼──────────┤
+│              iterations │        1 │        0 │
+├─────────────────────────┼──────────┼──────────┤
+│                requests │        6 │        0 │
+├─────────────────────────┼──────────┼──────────┤
+│            test-scripts │        6 │        0 │
+├─────────────────────────┼──────────┼──────────┤
+│      prerequest-scripts │        0 │        0 │
+├─────────────────────────┼──────────┼──────────┤
+│              assertions │        9 │        0 │
+├─────────────────────────┴──────────┴──────────┤
+│ total run duration: 283ms                     │
+├───────────────────────────────────────────────┤
+│ total data received: 53B (approx)             │
+├───────────────────────────────────────────────┤
+│ average response time: 6ms                    │
+└───────────────────────────────────────────────┘
+```
+
+To list all available options:
+
+```bash
+$ ./bin/run.sh
+...
+...
+run.sh:usage: [-cd Coredata] | [-md Metadata] | [-co Command] | [-sn SupportNotification] | [-ss SupportScheduler] | [-dv DeviceVirtual] | [-asc AppServiceConfigurable] | [-sec securityTest] | [-all All]
+```
+
+### Present Test Result Using the Allure Framework
+
+Allure is based on standard xUnit results output. Once we have finished running the Newman script, the built-in JUnit reporter outputs a summary of the collection run to a JUnit compatible XML file. (Path: /blackbox-testing/bin/testResult/)
+
+The simplest way to use the service is through the Allure Docker service. For exhaustive documentation, see https://github.com/fescobar/allure-docker-service. 
+
+Alternatively, install Allure as described in https://docs.qameta.io/allure/#_get_started
+
+Install and start Allure Docker service
+
+```bash
+docker pull frankescobar/allure-docker-service
+docker run -p 4040:4040 -p 5050:5050 -e CHECK_RESULTS_EVERY_SECONDS=3 -e KEEP_HISTORY="TRUE" -v /tmp/testResult:/app/allure-results frankescobar/allure-docker-service
+```
+
+This mounts the test results and checks for updates every 3 seconds. Now point your browser to http://localhost:4040 where you can review the report.
+
```

Comment:
Maybe only use port 5050, and use http://localhost:5050/latest-report to review the report.
(See https://github.com/fescobar/allure-docker-service#port-4040-deprecated)

Reply:
Additionally, is database required for app-service-configurable using blackbox-tests profile? Currently, the Database_Host is mongo.
https://github.com/edgexfoundry/blackbox-testing/blob/80f83ffa89dc1e00faabdb4d539fe28305369c67/docker-compose-test-tools.yml#L46-L48

Reply:
> Additionally, is database required for app-service-configurable using blackbox-tests profile?

Nope, DB not used. Removed the Database overrides.

Reply:
> Maybe only use port 5050,

Out of scope for this PR, but simple change to include... ;-)